app/controllers/admin/featured_slides_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Admin
+  class FeaturedSlidesController < Admin::BaseController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable '@featured_slide' [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->